curl.go

```diff
@@ -90,7 +96,29 @@ func main() {
 }
 
 func run(ctx context.Context) error {
-	flags.Parse(os.Args[1:])
+	cArgs := make([]string, 0)
+	flags.ParseAll(os.Args[1:], func(flag *pflag.Flag, value string) error {
+		if flag.Name == "silent" {
+			return nil // --silent is added later to all curl arguments so don't add here
+		}
+
+		// if it's a curl flag, save the full name & value to pass as arguments later
+		found := cflags.Lookup(flag.Name)
+		if found != nil {
+			if flag.Value.Type() == "bool" {
+				cArgs = append(cArgs, "--"+flag.Name)
+			} else {
+				cArgs = append(cArgs, "--"+flag.Name)
+				cArgs = append(cArgs, value)
+			}
+		}
+
+		err := flags.Set(flag.Name, value)
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
Would this be an equivalent but simpler expression of the same code?
```suggestion
		return flags.Set(flag.Name, value)
```

Reply:
RIght you are!